wai-aria/role/list-roles.html

```diff
@@ -18,12 +18,9 @@
   <div role="listitem" data-testname="last simple listitem" data-expectedrole="listitem" class="ex">x</div>
 </div>
 
-<div role="listitem" data-testname="orphan div with listitem role" class="ex-generic">x</div>
```

Comment:
Good catch that this one was not running due to the lack of `verifyGenericRoleByselector();`!

Reply:
Just a reminder that your prior PR fixed (in a new file) the lack of `verifyGenericRoleByselector()` in list-roles.html and tab-roles.html... You can add those back in here if you like, or we can handle it in:
- https://github.com/web-platform-tests/interop-accessibility/issues/111. 

Either is fine with me. Thanks,

Reply:
Definitely, thank you! I think I'll handle it in https://github.com/web-platform-tests/interop-accessibility/issues/111, just to separate things a bit between (1) resolution of spec question and (2) enabling tests.

Reply:
Addressed (2) in the PR here: https://github.com/web-platform-tests/wpt/pull/45174